packages/foundation/index.d.ts

```diff
@@ -73,6 +73,14 @@ export declare function findConfigurationFileRecursive (
   schemas?: string | string[],
   suffixes?: string | string[]
 ): Promise<string | null>
+export declare function findRuntimeConfigurationFile (
+  logger: Logger,
+  root: string,
+  configurationFile: string,
+  fallback?: boolean,
+  throwOnError?: boolean,
+  verifyPackages?: boolean
+): Promise<string | null>
```

Comment:
why was this added?

Reply:
The method was already there, it is used by CLI machinery in Watt.